zetaclient/metrics/telemetry.go

```diff
@@ -251,6 +263,13 @@ func (t *TelemetryServer) hotKeyFeeBurnRate(w http.ResponseWriter, _ *http.Reque
 	fmt.Fprintf(w, "%v", t.HotKeyBurnRate.GetBurnRate())
 }
 
+func (t *TelemetryServer) connectedPeersHandler(w http.ResponseWriter, _ *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	peers := t.GetConnectedPeers()
+	data, _ := json.Marshal(peers)
```

Comment:
We should check error here

Reply:
e927e2a70047e72a707d0697e73e5285c4850ced